.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,57 @@
+# Contributions
+
+ClassicPress is a volunteer-driven open source project that thrives when we have multiple contributors working towards a common goal.
+
+Before contributing to this repository, please read our [Democracy page](https://www.classicpress.net/democracy/) to understand how we make decisions about what is included in ClassicPress, and our [roadmap](https://www.classicpress.net/roadmap/) to see what's already planned for the next couple of releases.
+
+The **petitions process** mentioned in that document is very important to help us prioritize new features based on the needs of our users. There are some [exceptions](https://www.classicpress.net/democracy/#democracy-exceptions) to this process for minor changes and bugfixes, but generally speaking it is a good idea to create an issue or a forum thread, or ask in Slack, before undertaking a change.
+
+Also, please be sure to follow our [code of conduct](https://www.classicpress.net/democracy/#democracy-conduct) in all interactions with ClassicPress community members.
+
+## Communication channels
+
+We encourage you to join and ask any questions you have about contributing.
+
+- [Slack](https://www.classicpress.net/join-slack/) - great for real-time chat, posting screenshots and asking questions about anything you're stuck on. Or just socializing.
+- [Petitions website](https://petitions.classicpress.net/) - for proposing new features or major changes for consideration by the community.
+- [GitHub issues](https://github.com/ClassicPress/ClassicPress/issues) - for proposing or discussing bugfixes, or minor improvements. Generally it is a good idea to create a petition for anything that may take a significant amount of time or that may have backwards compatibility implications.
+- [Forums](https://forums.classicpress.net/) - a lot of good info and discussion here too
```

Comment:
Our forums are a great resource, I want to say something better than "a lot of good info and discussion here too". In what situations would someone want to make a forum thread rather than using one of the other communication channels?